.github/workflows/generate_manpage.yml

```
@@ -19,7 +19,13 @@ jobs:
           python-version: 3.x
 
       - name: Install dependencies
-        run: pip install argparse-manpage
+        run: |
+          pip install argparse-manpage six pexpect rpm
```

Okay, so the `rpm` package here is not doing what we think it is doing... We are expecting to install the python3 rpm bindings for python, but in turn, we are actually installing this one here: https://pypi.org/project/rpm/

Which is a package maintained by packit (https://github.com/packit/rpm-shim).

I will say that we can remove this package from pip and rely on python3-rpm binary being installed on the next step. 

```suggestion
          pip install argparse-manpage six pexpect
```

---

Also, the python3-rpm should take care of installing the desired library we need. 

```bash
[r0x0d@fedora ~]$ rpm -ql python3-rpm
/usr/lib/.build-id
/usr/lib/.build-id/dc
/usr/lib/.build-id/dc/700247b4d4b61b9b0da932787b19546ff6e80e
/usr/lib64/python3.11/site-packages/rpm-4.18.1-py3.11.egg-info
/usr/lib64/python3.11/site-packages/rpm/__init__.py
/usr/lib64/python3.11/site-packages/rpm/__pycache__
/usr/lib64/python3.11/site-packages/rpm/__pycache__/__init__.cpython-311.opt-1.pyc
/usr/lib64/python3.11/site-packages/rpm/__pycache__/__init__.cpython-311.pyc
/usr/lib64/python3.11/site-packages/rpm/__pycache__/transaction.cpython-311.opt-1.pyc
/usr/lib64/python3.11/site-packages/rpm/__pycache__/transaction.cpython-311.pyc
/usr/lib64/python3.11/site-packages/rpm/_rpm.so
/usr/lib64/python3.11/site-packages/rpm/transaction.py
```

---

Ok, cool thank you! I was just looking into python3-rpm, I will go ahead and take rpm out of the install command!